src/databasetest/DatabaseTest.java

```diff
@@ -0,0 +1,290 @@
+3: for value As Integer = 0 To 5
+4: for counter in 1..5
+5: for(i in 1:10)
+6: for n = 1:5 
+7: for count := 1 to 100 do 
+
+WHILE LOOP
+0: other
+1: while (counter > 1)
+
+ARRAYS (size, first, last) )
+0: other
+1: name.length, 0, name.length-1
+2: array[size], 0, size
+3: len(name), 0, len(name)-1
+4: UBound(name)-LBound(name)+1, LBounf(name), UBound(name)
+5: count($name), 0, count($name)-1
+6: scalar(@name), $[, $#name
+7: [name count], 0, [name count] - 1
+8: length(name), 1, end
+9: #name, 1, #name
+
+OPERATORS
+0: other
+1: () [] -> . ! ~ ++ -- + - * & / % << >> < <= > <= == != ^ | && || ?: = += -= *= /= %= &= ^= |= <<= >>=
+2: ^ + - * / \ MOD = <> > < >= <= AND OR NOT XOR ANDALSO ORELSE ISFALSE ISTRUE & | ^ ~ = += -= *= /= \= ^= <<= >>= &=
+3: ] ** ++ -- ~ @! * / % + - . << >> < <= > >= == != === !== <> <=> & ^ | && || ?? ?: = += -= *= **= /= .= %= &= |= ^= <<= >>=
+4: -> ++ -- ** ! ~ \ + - . =~ !~ * / % < > <= >= == != <=> ~~ & | ^ && || '
+5: + - * / % ** == != > < >= <= <=> === .eql? equal? = += -= *= /= %= **= & | ^ ~ << >>  and or && || ! not
+6: + - * / %% %/% ^ > < == <= >= != & | ! && || <- -> : %in% %*%
+7: + - * / ** = != <> ~= > < >= <= LIKE BETWEEN IN IS_NULL and or not ** + - * / ||
+8: not # - .. * / % + - < > <= >= == ~= and or
+
+USES,PARADIGMS
+ */
+public class DatabaseTest {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) throws ClassNotFoundException {
+        Connect();
+    }
+
+    public static void Connect() {
+        //Database Info
+        String host = "jdbc:mysql://localhost:3306/mydb";
+        String Uname = "root";
+        String Pword = "password";
+
+        try {
+            //load driver
+            Class.forName("com.mysql.jdbc.Driver");
+
+            //Connect to database
+            Connection con = DriverManager.getConnection(host, Uname, Pword);
+
+            //Create statement
+            Statement stmt = con.createStatement();
+            
+            //create empty table
+            String dummy = "CREATE TABLE `percentages` (`Dummy` varchar(69))";
+            
+            stmt.executeUpdate(dummy);
+            
+            //call Load CLass
+            load(stmt, con);
+
+        } catch (SQLException | ClassNotFoundException err) {
+            System.out.println(err.getMessage());
+        }
+    }
+
+    public static void load(Statement stmt, Connection con) throws SQLException {
+        //initailise variables
+        String[][] CompArray;
+        String Test = "*";
+        String Condition = "";
+
+        String Get = "SELECT " + Test + " FROM mydb.languages " + Condition;
+
+        //Execute Query on statement
+        ResultSet rs = stmt.executeQuery(Get);
+
+        int DatabaseSize = 0;
+        int ColumnNum = 15;
+
+        if (rs.last()) {
+            DatabaseSize = rs.getRow();
+            rs.beforeFirst();
+        }
+
+        CompArray = new String[DatabaseSize][ColumnNum];

```

Comment:
Below this code you set DatabaseSize = 0, followed by [0][number] for your array  for each rs.next(). Does this means you only need a 1D array?